### PR TITLE
Test histogram

### LINKED ISSUE #19: Add tests using testthat for Histogram

### PR DESCRIPTION
**Pull request by shyan0903**

### CODE REVIEW

**Review by miyer26 — APPROVED** [review]

File: R/histogram.R [hunk 1]
```diff
@@ -1,17 +1,86 @@
...
+  # check to ensure y is in quotes
+  if (
+    typeof(substitute(y)) == typeof(as.list(quote(symbol))[[1]])
+  ) {
+    stop("The aggregation function assigned to 'y' must be in quotes.")
+  }
+
+  # check to ensure y is in the correct format and has the supported value
+  if (!startsWith(y, "..") || !endsWith(y, "..") ) {
+    stop("Value assigned to 'y' should start with '..' and end with '..'.")
```
Comment (miyer26) [inline R/histogram.R]: Just curious Irene, isn't this redundant as you are already checking to see if the inout for y exists in the `aggregation functions` list?

Comment (shyan0903) [reply R/histogram.R]: Oh yeah you are right! Thank you for catching this!

**Review by ruben1dlg — APPROVED** [review]
Great work! Looks very good! 😀

**Comment by iamMoid** [comment]
@shyan0903 As discussed, please update this branch from main to bring in your changes to the histogram.R and resolve any merge conflicts.

**Pull request merged**